### DISCUSSION TITLE
[OPINION] Main Menu should be returned to its original place.

### DISCUSSION THREAD

**Discussion by MerryPanda** [phcode-dev/phoenix#1533] [General]
The Main menu belongs to the whole program so it should be placed to where all programs usually place their Main menus (to the left top as in the original Brackets). It will feel more intuitive.

**Comment by abose** [comment]
Thanks for the feedback. We are doing quite a few ui changes over the coming months, but it will not be drastic departures from Brackets original UI, but an extension. For the above styling comment, ill let this issue open for others to comment on their take.
